pkg/cloudprovider/cloud.go

```diff
@@ -37,6 +37,8 @@ type Interface interface {
 	Clusters() (Clusters, bool)
 	// Routes returns a routes interface along with whether the interface is supported.
 	Routes() (Routes, bool)
+	// Volumes returns a volumes interface along with whether the interface is supported.
+	Volumes() (Volumes, bool)
```

Comment:
What if a cloud provider supports more than one kind of Volume?  This feels like the wrong factoring to me.